### PR TITLE
Add ManifestWorkReplicaSet doc

### PR DESCRIPTION
**Pull request by serngawy**

### CODE REVIEW

File: content/en/concepts/manifestworkreplicaset.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+weight: 3
+---
+
+<!-- spellchecker-disable -->
+
+{{< toc >}}
+
+<!-- spellchecker-enable -->
+
+## What is `ManifestWorkReplicaSet`
```
Comment (qiujian16) [inline content/en/concepts/manifestworkreplicaset.md:12]: we should indicate that this is a alpha feature now and need to be enabled by the feature gate.

Comment (serngawy) [reply content/en/concepts/manifestworkreplicaset.md:12]: ops, sure will add it

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**